From 695afbcc37c997fbe657d4b87e7e2d5976732f84 Mon Sep 17 00:00:00 2001
From: Archit Taneja <archit@ti.com>
Date: Fri, 16 May 2014 12:58:06 +0530
Subject: [PATCH 0939/1587] ARM: OMAP2+: Add detection of dra7-dss

Add detection of dra7-dss nodes so that the related devices get
initialized.

Signed-off-by: Archit Taneja <archit@ti.com>
Signed-off-by: Darren Etheridge <detheridge@ti.com>
[zou: Original patch taken from
ti-sdk-am335x-evm-08.00.00.00-Linux-x86-Install.bin]
Signed-off-by: Cao Zou <cao.zou@windriver.com>
---
 arch/arm/mach-omap2/display.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-omap2/display.c b/arch/arm/mach-omap2/display.c
index 2f840f9..57a49ca 100644
--- a/arch/arm/mach-omap2/display.c
+++ b/arch/arm/mach-omap2/display.c
@@ -574,6 +574,10 @@ struct device_node * __init omapdss_find_dss_of_node(void)
 	if (node)
 		return node;
 
+	node = of_find_compatible_node(NULL, NULL, "ti,dra7-dss");
+	if (node)
+		return node;
+
 	return NULL;
 }
 
-- 
1.7.5.4

